doc/source/install/windows.rst

```diff
@@ -9,14 +9,43 @@ Mroonga binary for Windows is provided with MariaDB binary because
 <https://github.com/mroonga/mroonga/tree/main/packages/source/patches>`_
 are needed for building Mroonga for Windows.
 
-Zip
----
+Install
+-------
+
+Download and extract the zip file
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Download zip file and extract it. You need to choose a zip for your
 environment:
 
   * |mroonga_mariadb_windows_package_link|
```

Comment:
This is not related to this change but could you fix markup problem too?

```suggestion
* |mroonga_mariadb_windows_package_link|
```

In `.rst`, we can't use indent for the top-level list. It's a markup of "quote" and "list".